Hakawai/Mentions/_HKWMentionsStartDetectionStateMachine.h

```diff
@@ -125,4 +138,9 @@
  */
 -(void) resetStateUsingString:(NSString *)string;
 
+/*!
+ Return characters after given location till whitespace is encountered.
```

Comment:
add arg comments and an example of how this is used in the code

Reply:
Added doc comment